packages/travix-ui-kit/components/stepper/stepper.scss

```diff
@@ -0,0 +1,100 @@
+.ui-stepper {
+  display: flex;
+  height: 39px;
+  justify-content: space-between;
+
+  .ui-stepper__button {
```

Comment:
All selectors down below are still nested inside `.ui-stepper`. These styles should not depend on the parent.